charts/kube-prometheus-stack/values.yaml

```diff
@@ -2162,6 +2190,19 @@ nodeExporter:
 prometheus-node-exporter:
   namespaceOverride: ""
   fullnameOverride: "node-exporter"
+
+  image:
+    registry: quay.io
+    repository: prometheus/node-exporter
+    tag: ""
+  resources:
+    limits:
+      cpu: 250m
```

Comment:
@junotx what's the default limits in 3.5 for node exporter to avoid throttling warning?

Reply:
may refer to https://github.com/kubesphere/kse-installer/blob/kse-3.x/roles/ks-monitor/templates/node-exporter-daemonset.yaml.j2#L46

Reply:
fixed

Reply:
make sure limits for other monitoring components remain consistent with v3.5